AmplifyPlugins/Core/AWSPluginsCore/Model/Support/ModelSchema+GraphQL.swift

```diff
@@ -11,9 +11,26 @@ import Foundation
 /// Extension that adds GraphQL specific utilities to `ModelSchema`.
 extension ModelSchema {
 
-    /// The GraphQL name of the schema.
-    var graphQLName: String {
-        name
+    /// The GraphQL directive name translated from the GraphQL operation and model schema data
+    func graphQLName(operationSubType: GraphQLOperationSubType) -> String {
+        let graphQLName: String
+        switch operationSubType {
+        case .mutation(let mutationType):
+            graphQLName = mutationType.rawValue + name
+        case .query(let queryType):
+            switch queryType {
+            case .list:
+                graphQLName = queryType.rawValue + name + "s"
```

Comment:
make a String helper method called `pluralize` (I actually thought we already had one somewhere), to centralize and abstract how we pluralize strings